test/helpers/policygen/models.go

```diff
@@ -0,0 +1,664 @@
+package policygen
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"net"
+	"os"
+	"reflect"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/cilium/cilium/test/helpers"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	log "github.com/sirupsen/logrus"
+)
+
+// ConnTestSpec Connectivy Test Specification. This is a struct to use in
+// PolicyTest to determine the result of the TestCase.
+type ConnTestSpec struct {
+	HTTP        ResultType
+	HTTPPrivate ResultType
+	Ping        ResultType
+	UDP         ResultType
+}
+
+// GetField method to retrieve the value of any type of the struct.
+// It is used by `TestSpec` to created expected results
+func (conn *ConnTestSpec) GetField(field string) ResultType {
```

Comment:
I think it should be a switch statement because the fields are already constants defined in `./const.go`, something like:
```golang
func (conn *ConnTestSpec) GetField(field fieldTypes) ResultType {
   switch field{
   case HTTPPrivate:
    return conn.HTTPrivate
    case ...
  } 
}
```
(and it should also by a `fieldTypes` in `const.go`)